### PR TITLE
Added quiz editor component

### PR DESCRIPTION
**Pull request by ronan-f**

### CODE REVIEW

**Review by wongvincent — APPROVED** [review]

File: web-components/d2l-activity-editor.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 import 'd2l-activities/components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor.js';
+import 'd2l-activities/components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-editor.js';
```
Comment (dlockhart) [inline web-components/d2l-activity-editor.js:2]: Should these be separate imports? Mainly: do they both always appear together, or are there scenarios where only one or the other are needed?

Comment (ronan-f) [reply web-components/d2l-activity-editor.js:2]: @dlockhart I would say there are scenarios when only one or the other is needed. For example, if an instructor is creating a quiz then we probably don't want to import the assignment editor component as well.

In that case should they have their own files with separate imports?

Comment (wongvincent) [reply web-components/d2l-activity-editor.js:2]: Hmm.. I would think only one or the other would be needed at any one time. Either an assignment or a quiz editor would be on the page and not both.

Are you suggesting they be in separate files in `/web-components`?

Comment (dlockhart) [reply web-components/d2l-activity-editor.js:2]: Yeah sounds like they should be separate. Now that things are unbundled, you actually don't need anything here in `/web-components`. You can just create entries in `rollup-wc.config.js` that point directly into `node_modules`, similar [to this example](https://github.com/Brightspace/brightspace-integration/blob/master/rollup/rollup-wc.config.js#L77).

Comment (dlockhart) [reply web-components/d2l-activity-editor.js:2]: If you want a non-breaking way to do this:
1. Add individual entries to `rollup-wc.config.js` for each of these two files
2. Wait for BSI to deploy
3. In the LMS, switch the `ActivityEditorImport.codegen` file to point to the new location of the editor. Aside: there are actually 2 copies of that same codegen file, which isn't great... ideally there's a single definition for the asset and it gets loaded from a single view's renderer and it's the view itself that gets used in multiple places.
4. Once that merges, you can remove this file and the entry for it in `rollup-wc.config.js`.

**Pull request merged**